vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/model/Breadcrumbs.java

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See {@literal <https://vaadin.com/commercial-license-and-service-terms>} for the full
+ * license.
+ */
+package com.vaadin.flow.component.charts.model;
+
+import com.vaadin.flow.component.charts.model.style.ButtonTheme;
+import com.vaadin.flow.component.charts.model.style.Style;
+
+/**
+ * Options for the breadcrumbs, the navigation at the top leading the way up
+ * through the drilldown levels.
+ */
+public class Breadcrumbs extends AbstractConfigurationObject {
+
+    private Number buttonSpacing;
+    private ButtonTheme buttonTheme;
+    private Boolean floating;
+    private String format;
+    private String _fn_formatter;
+    private BreadcrumbsPosition position;
+    private ButtonRelativeTo relativeTo;
+    private Boolean rtl;
+    private Separator separator;
+    private Boolean showFullPath;
+    private Style style;
+    private Boolean useHTML;
+    private Number zIndex;
+
+    /**
+     * @return The space between individual breadcrumb buttons in pixels.
+     */
+    public Number getButtonSpacing() {
+        return buttonSpacing;
+    }
+
+    /**
+     * Sets the space between individual breadcrumb buttons in pixels.
+     */
+    public void setButtonSpacing(Number buttonSpacing) {
+        this.buttonSpacing = buttonSpacing;
+    }
+
+    /**
+     * @return Theme options for the breadcrumb buttons.
+     */
+    public ButtonTheme getButtonTheme() {
+        if (buttonTheme == null) {
+            buttonTheme = new ButtonTheme();
```

Comment:
The default button has a width of `32`, which doesn't work very well here, as it results in the breadcrumbs showing on top of each other.